src/lib/PreviewUI.js

```diff
@@ -299,6 +310,10 @@ class PreviewUI {
      * @return {void}
      */
     showNotification(message, buttonText) {
+        if (!this.notification) {
+            this.setupNotification();
+        }
```

Comment:
Good idea moving it here!

Reply:
Easiest fix would be:

```
if (!this.notification) {
    this.setupNotification();
}

if (this.notification) {
     this.showNotification();
}
```

But, I'm not a huge fan of how that reads... Open to other suggestions.

Reply:
cleaned it up a bit. Thoughts?